lib/bender/capistrano/tasks/robots.rake

```diff
@@ -0,0 +1,16 @@
+namespace :deploy do
```

Comment:
If this is in the `deploy` namespace, why not have it in the `deploy.rake`? To be a little bit more explicit about what is run where. So you don't have to guess the order of things running by looking into every file.